.gitlab-ci.yml

```diff
@@ -34,3 +34,24 @@ test:unit:
     - nats
   variables:
     DEVICECONNECT_MONGO_URL: "mongodb://mongo"
+
+test:acceptance:
+  stage: test
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  tags:
+    - docker
+  image: docker:19.03.5-dind
+  before_script:
+    - apk add docker-compose make
+  script:
+    - make acceptance-tests
+  after_script:
+    - make acceptance-tests-logs
+    - make acceptance-tests-down
```

Comment:
Very nice! :+1: 

If we standardize this in couple more repos down the road, it will make sense to extend the template to support backed repos with Makefile.